### PR TITLE
Revert "[node] Revert 20.13 changes that were missing version bump (#…

### PR DESCRIPTION
**Pull request by jakebailey**
Reverts https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69716 and bumps the version

### CODE REVIEW

**Comment by jakebailey** [comment]
waiting for https://github.com/microsoft/DefinitelyTyped-tools/actions/runs/9322967998

File: types/node/fs.d.ts [hunk 1]
```diff
@@ -128,6 +128,7 @@ declare module "fs" {
      *   ctime: Mon, 10 Oct 2011 23:24:11 GMT,
      *   birthtime: Mon, 10 Oct 2011 23:24:11 GMT }
      * ```
+     * @deprecated Since v20.13.0. Public constructor is deprecated.
```
Comment (codershiba) [inline types/node/fs.d.ts:131]:
```suggestion
```

Comment (codershiba) [reply types/node/fs.d.ts:131]: https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/69713

Comment (jakebailey) [reply types/node/fs.d.ts:131]: Uh, sure

**Review by ghost — APPROVED** [review]

**Pull request merged**